README.md

```diff
@@ -261,6 +263,8 @@ All props are optional.
 
 **tag** { String }: The HTML tag for this element. Default: `'span'`.
 
+**component** { Object }: A component instance to use instead of the tag.
+
 *Any additional props (e.g. `id`, `className`, `data-whatever`) are passed directly to the HTML element.*
```

Comment:
Let's change this line, too: `passed directly to the HTML element or custom component`.